src/Queue/QueueBase.php

```diff
@@ -117,7 +117,7 @@ protected function getQueue(AMQPChannel $channel, array $options = []) {
 
     // Allow modules to override queue settings.
     if (isset($queue_info[$this->name])) {
-      $queue_options += $queue_info[$this->name];
```

Comment:
Since I did the error initially and you did another fixing mine, this probably means a test is in order to catch future regressions.